lca-ai-stack/deployment/lca-ai-stack.yaml

```diff
@@ -389,6 +393,12 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: RuleRetention
+            Status: Enabled
+            Prefix: ""
+            RecordingsBucketRetentionDays: !Ref RecordingsBucketRetentionDays
```

Comment:
@jdrmmaster This is broken code - maybe the result of a search/replace without follow-up testing?  It causes the stack to fail to deploy. 
`RecordingsBucketRetentionDays:` is not a valid  rule parameter for a Lifecycle Configuration rule. The rule key must be `ExpirationInDays`.  
To avoid further iterations we'll accept the PR and make the change before release. 
![image](https://user-images.githubusercontent.com/10953374/233649525-ae7e91fa-b95f-469d-aa64-65d853a0bfcc.png)